css/styles.css

```diff
@@ -34,12 +33,27 @@ header {
   margin-right: 100px;
 }
 
+.btn-pos {
+  position: absolute !important;
+  bottom: 0 !important;
+  width: 100% !important;
+}
+
+.rel-pos {
+  position: relative !important;
+}
+
+.nav {
+  display: none;
+}
+
 .headline_section {
```

Comment:
- [ ] Please I highly suggest adding all social media icons per the [FIGMA DESIGN](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=HhgzrHa7uTvTkdte-0)

_**REQUIRED FIGMA DESIGN**_

![Screenshot (193)](https://user-images.githubusercontent.com/78104544/229261869-52e56010-e951-476b-807f-b7c62a9598a9.png)

_**YOUR DESIGN**_

![Screenshot (192)](https://user-images.githubusercontent.com/78104544/229261892-692e073f-8079-4a3d-bf23-89ba2e630bab.jpg)